go/vt/sqlparser/ast.go

```diff
@@ -3083,6 +3090,7 @@ func (*ExtractValueExpr) iCallable()                   {}
 func (*UpdateXMLExpr) iCallable()                      {}
 func (*PerformanceSchemaFuncExpr) iCallable()          {}
 func (*GTIDFuncExpr) iCallable()                       {}
+func (*PointExpr) iCallable()                              {}
```

Comment:
I suspect the static linter checks fail on this one or the previous as they are not lined out. Please run `gofmt` for these as the CI check mentions.